frameworks/PHP/peachpie/Website/db.php

```diff
@@ -0,0 +1,24 @@
+<?php
+//
+// Database Test
+//
+
+function db() {
+  
+  // Set content type
+  header("Content-type: application/json");
+
+  // Database connection (TODO: When it works, use PDO instead)
+  $link = mysql_pconnect('tfb-database', 'benchmarkdbuser', 'benchmarkdbpass');
+  mysql_select_db('hello_world', $link);
+
+  $id = mt_rand(1, 10000);
+  $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+
+  // Store result in array.
+  echo json_encode( ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)] );
```

Comment:
Must be read the id and the randomNumber.
https://github.com/TechEmpower/FrameworkBenchmarks/issues/5167#issuecomment-544679539

Reply:
thank you